sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py

```diff
@@ -189,68 +114,9 @@ def test_output_same_order_as_input(self, client):
         for idx, doc in enumerate(response):
             self.assertEqual(str(idx + 1), doc.id)
 
-    @pytest.mark.playback_test_only
-    @GlobalTextAnalyticsAccountPreparer()
-    @TextAnalyticsClientPreparer(client_kwargs={
-        "text_analytics_account_key": "",
-    })
-    def test_empty_credential_class(self, client):
-        with self.assertRaises(ClientAuthenticationError):
-            response = client.begin_analyze_healthcare_entities(
-                ["This is written in English."],
-                polling_interval=self._interval()
-            ).result()
-
-    @pytest.mark.playback_test_only
-    @GlobalTextAnalyticsAccountPreparer()
-    @TextAnalyticsClientPreparer(client_kwargs={
-        "text_analytics_account_key": "xxxx",
-    })
-    def test_bad_credentials(self, client):
-        with self.assertRaises(ClientAuthenticationError):
-            response = client.begin_analyze_healthcare_entities(
-                ["This is written in English."],
-                polling_interval=self._interval()
-            )
-
-    @pytest.mark.playback_test_only
-    @GlobalTextAnalyticsAccountPreparer()
-    @TextAnalyticsClientPreparer()
-    def test_bad_document_input(self, client):
-        docs = "This is the wrong type"
-
-        with self.assertRaises(TypeError):
-            response = client.begin_analyze_healthcare_entities(docs, polling_interval=self._interval())
-
-    @GlobalTextAnalyticsAccountPreparer()
-    @TextAnalyticsClientPreparer()
-    def test_mixing_inputs(self, client):
-        docs = [
-            {"id": "1", "text": "Microsoft was founded by Bill Gates and Paul Allen."},
-            TextDocumentInput(id="2", text="I did not like the hotel we stayed at. It was too expensive."),
-            u"You cannot mix string input with the above inputs"
-        ]
-        with self.assertRaises(TypeError):
-            response = client.begin_analyze_healthcare_entities(docs, polling_interval=self._interval())
-
-    @GlobalTextAnalyticsAccountPreparer()
-    @TextAnalyticsClientPreparer()
-    def test_out_of_order_ids(self, client):
```

Comment:
We could consider keeping only this test (instead of the other one above testing the ordering) in order to also ensure that we are always sorting based on the input order rather than naively (i.e. alphanumerically).

Reply:
good point, i'll just get rid of the alphanumeric one then